docs/docfx.json

```diff
@@ -14,7 +14,7 @@
           "src": ".."
         }
       ],
-      "dest": "docs/api"
```

Comment:
Can you also do something so that it doesn't build docs of Sample projects? Those are not part of the API.

Reply:
Done. Each sample project is excluded manually in the docfx.json